### PR TITLE
 FS-2819: Redirect to dashboard when clicked back from submit

### PR DESCRIPTION
**Pull request by RamuniN**
https://digital.dclg.gov.uk/jira/browse/FS-2819

### Description
Added cache control, disable going back to tasklist from submit page instead redirect to dashboard

### CODE REVIEW

**Review by gio-karunakaran — APPROVED** [review]
LGTM

File: app/create_app.py [hunk 1]
```diff
@@ -211,6 +211,16 @@ def inject_content_urls():
...
 
+    @flask_app.after_request
+    def after_request(response):
+        if "Cache-Control" not in response.headers:
+            response.headers[
+                "Cache-Control"
+            ] = "no-cache, no-store, must-revalidate"
+            response.headers["Pragma"] = "no-cache"
+            response.headers["Expires"] = "0"
+        return response
```
Comment (tferns) [inline app/create_app.py:222]: IIRC didn't we recently remove this?  Or was that for assessment 🤔

File: app/default/application_routes.py [hunk 2]
```diff
@@ -131,15 +130,6 @@ def tasklist(application_id):
...
     application = get_application_data(application_id, as_dict=True)
-    account = get_account(account_id=application.account_id)
-    if application.status == ApplicationStatus.SUBMITTED.name:
-        with force_locale(application.language):
-            return render_template(
-                "application_submitted.html",
-                application_id=application.id,
-                application_reference=application.reference,
-                application_email=account.email,
-            )
```
Comment (tferns) [inline app/default/application_routes.py:142]: If I'm reading this right, we don't want to show the user the successful submission page anymore?

And instead just show them their dashboard?

Comment (tferns) [reply app/default/application_routes.py:142]: Oh wait.. this is on the tasklist route 🤔 

Okay I understand the fix now!  Strange.

**Review by tferns — APPROVED** [review]
LGTM 🚢

File: app/create_app.py (hunk 1 above)
Comment (RamuniN) [reply app/create_app.py:222]: cache-control was not implemented in front-end before

File: app/default/application_routes.py (hunk 2 above)
Comment (RamuniN) [reply app/default/application_routes.py:142]: Here we are preventing the user going back to tasklist page when clicked on back button from submitted page, instead redirect to dashboard

File: app/create_app.py (hunk 1 above)
Comment (tferns) [reply app/create_app.py:222]: Ah so it was assessment! thanks

**Pull request merged**